zerver/webhooks/gocd/doc.md

```diff
@@ -1,24 +1,33 @@
-Zulip supports integration with GoCD and can notify you of
-your build statuses.
+# Zulip GoCD integration
+
+Get GoCD notifications in Zulip!
+
+{start_tabs}
 
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
-1. Add the following to your `Config.XML` file.
+1. Download and install version **v0.0.6** of Sentry's [**WebHook Notifier    plugin**][gocd-webhook-link]. Please note that newer versions might not work
+   properly.
 
-    ```
-    <pipeline name="mypipeline">
-        <trackingtool link="<URL constructed above>" regex="##(\d+)"/>
-        ...
-    </pipeline>
-    ```
+1. In your GoCD server, go to **Admin>Server Configuration>Plugins**. Click
```

Comment:
Let's add spaces around the `>` here ... "**Admin > Server Configuration > Plugins**" for readability.